clippy_lints/src/ptr.rs

```diff
@@ -277,8 +277,6 @@ fn check_invalid_ptr_usage<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx Expr<'_>) {
             | sym::ptr_read_unaligned
             | sym::ptr_read_volatile
             | sym::ptr_replace
-            | sym::ptr_slice_from_raw_parts
```

Comment:
I feel like this should still lint this, just with checking the next usages (before the next reassignment) to ensure it's used. Because yeah, just linting the function ends up being incorrect. But this is a lot for just a single part of this lint so I don't think it should be added rn. Can you leave a comment about this?

Reply:
Done.